build/config/babel/bundler.js

```diff
@@ -24,17 +24,14 @@ function getPreamble(babel) {
   var t = babel.types;
   var body = [];
   body.push(
-    t.expressionStatement(
-      t.literal('use strict')));
```

Comment:
If you put it at the top of the file, all scripts used by the app which included the bundled file will be run in strict mode.  The strict mode is turned on globally.  It's much safer to put `'use strict';` inside of the IIFE to turn it on just for the scope of the IIFE.